fetch/src/2_process.r

```diff
@@ -0,0 +1,36 @@
+process_data <- function() {
+  #Grab the data
+  mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
```

Comment:
Here, `project_output_dir` will need to be defined as `1_fetch/out`, but note that when you use it to save the `eval_data` to csv, below, you'll actually want to refer to a different directory -- since we're now in the `2_process` phase, the `model_summary_results.csv` should be saved to `2_process/out`, as should the `model_diagnostic_text.txt`.